backend/internal/supabase/migrations/20240917002108_base_schema.sql

```diff
@@ -60,4 +60,38 @@ create table if not exists review (
     comment text,
     created_at timestamp with time zone default now(),
     updated_at timestamp with time zone default now()
+);
+
+create table if not exists follower (
+    follower_id uuid not null references "user"(id),
+    followee_id uuid not null references "user"(id),
+    primary key (follower_id, followee_id)
+);
+
+create table if not exists playlist (
+    playlist_id serial primary key,
```

Comment:
let's call this `id` to match the other tables and so that your foreign key on line 80 doesn't break